scss/modules/_filters.scss

```diff
@@ -145,6 +155,39 @@
     height: auto;
   }
 
+  .combo--filter--mini {
```

Comment:
i think it's a case that it would make sense in either folder. however it seems `filters.scss` contains most of the interaction classes and styles now